src/MediaWiki/Hooks/SkinTemplateNavigation.php

```diff
@@ -45,6 +45,7 @@ public function __construct( SkinTemplate &$skinTemplate, array &$links ) {
 	public function process() {
 
 		if ( $this->skinTemplate->getUser()->isAllowed( 'purge' ) ) {
+			$skin->getOutput()->addModules( 'ext.smw.purge' );
```

Comment:
If I had a thorough look at the rest of the function `$this->skinTemplate->getOutput()...` was not far away from being figured out. :|